includes/core-changes/cryptography/5.0/cryptography-apis-not-supported-on-blazor-webassembly.md

```diff
@@ -0,0 +1,47 @@
+### System.Security.Cryptography APIs not supported on Blazor WebAssembly
+
+<xref:System.Security.Cryptography> APIs throw a <xref:System.PlatformNotSupportedException> at run time when run on a browser.
+
+#### Change description
+
+In previous .NET versions, the <xref:System.Security.Cryptography> APIs aren't available to Blazor WebAssembly apps. Starting in .NET 5.0, Blazor WebAssembly apps target the full .NET 5 API surface area, however, not all .NET 5 APIs are supported due to browser sandbox constraints. In .NET 5.0 and later versions, the <xref:System.Security.Cryptography> APIs throw a <xref:System.PlatformNotSupportedException> when running on WebAssembly.
```

Comment:
@steveisok Is the first sentence a correct statement? Or were the APIs available but just didn't throw the exception?

"In previous .NET versions, the <xref:System.Security.Cryptography> APIs aren't available to Blazor WebAssembly apps."